README.rst

```diff
@@ -41,6 +41,9 @@ geoscientific data (points, curve, surface, 2D and 3D grids) in
 ``*.geoh5`` file format. Users will be able to directly leverage the powerful visualization
 capabilities of `Geoscience ANALYST <https://mirageoscience.com/mining-industry-software/geoscience-analyst/>`_.
 
+.. contents:: Table of Contents
+   :local:
+   :depth: 3
```

Comment:
Need to update the link to the docs.

Reply:
Also update the Python supported (3.10, 3.11, 3.12)

Reply:
actually, reference to doc needs to be fixed in all the repos.
Seach for "https://(.*).readthedocs.io" and replace by "mirageoscience-\1.readthedocs-hosted.com"